lib/rockauth/configuration.rb

```diff
@@ -38,6 +39,9 @@ def resource_owner_class
 
     config.generate_active_admin_resources = nil
     config.active_admin_menu_name = 'Authentication'
+    config.password_reset_token_time_to_live = 24.hours
+    config.email_from = 'change-me-in-config-initializers-rockauth-rb@please-change-me.example'
+    config.forgot_password_always_successful = false
```

Comment:
More secure yes, but rockauth is fundamentally designed to run the authentication for social networking APIs. In which I think user frustration is more of a concern than people trying every email address to see if an account exists.